src/registries/chart_types.ts

```diff
@@ -175,3 +178,150 @@ chartComponentRegistry.add("gauge", GaugeChartComponent);
 chartComponentRegistry.add("scatter", ChartJsComponent);
 chartComponentRegistry.add("scorecard", ScorecardChartComponent);
 chartComponentRegistry.add("waterfall", ChartJsComponent);
+chartComponentRegistry.add("pyramid", ChartJsComponent);
+
+type ChartUICategory = "line" | "bar" | "column" | "pie" | "misc";
+
+export const chartCategories: Record<ChartUICategory, string> = {
+  line: _t("Line"),
+  column: _t("Column"),
+  bar: _t("Bar"),
+  pie: _t("Pie"),
+  misc: _t("Miscellaneous"),
+};
+
+export interface ChartUIInfo {
+  /** Type shown in the chart side panel */
+  chartUIType: string;
```

Comment:
what's different between chartType and charUIType ?

Reply:
I renamed `chartUIType` to `chartSubtype` to try to be clearer. 

The idea is to have subtypes, that are displayed as different chart types in the side panel, but that are the same type with different options in the model. Eg. bar chart/stacked bar chart, pie chart/doughnut chart, line chart/area chart, etc..